en_us.lang

```diff
@@ -367,4 +367,4 @@ gui.xaero_entity_radar_type=Type:
 gui.xaero_entity_radar_color=Color:
 gui.xaero_entity_radar_visiblity=Visibility:
 gui.xaero_entity_radar_names=Names:
-gui.xaero_box_entity_radar_names=Render entity names under the dots. \n \n Option "Player List" - render while the "List Players" key binding is pressed (TAB by default).
+gui.xaero_box_entity_radar_names=Render entity names under the dots. \n \n Option "Player List" - render when the "List Players" key binding (TAB by default) is pressed.
```

Comment:
Why is "when" better?

Reply:
AFAIK "while" should be used with continuous tenses, and "the List Players key binding (TAB by default) is pressed" is not one of them. But if you definitely think that "while" would be better here, please let me know, and I'll restore it.